src/handbook/customer/marketing/messaging.md

```diff
@@ -23,10 +23,18 @@ FlowFuse appeals to a broad range of customers, many of which do not fall into
 our 'ICP'. That's not to say FlowFuse shouldn't cater to this audience, but
 does 
 
+1. Manufacturing or industrial company
+$500MM or up in revenue
+At least a 3 man IT team
+Established or desired digitalization strategy
+
+
 1. Manufacturing or industrial company
 1. $500MM or up in revenue
 1. At least a 3 man IT team
 1. Established Digitalization strategy
+   * Are looking to integrate many different machines, services, and systems
+   * Have one tool to connect, collect, visualize and interact with their operations
```

Comment:
```suggestion
   * Are using adjacent technologies, such as MQTT
   * Participating in Industry 4.0 Discord
```

Reply:
@MichaelBDavis are these must haves? Feels weird to include a discord in this list